### ISSUE TITLE
TypeError: Cannot read property 'charAt' of undefined

### ISSUE THREAD

**Issue by Bleser** [stscoundrel/harlaw#60]
```
TypeError: Cannot read property 'charAt' of undefined
    at /mnt/terro/PROJECT/GO/reChineseServer/cmd/utils/transform_dict/node_modules/.pnpm/harlaw@1.0.6/node_modules/harlaw/src/services/formatter.ts:60:30
...
```

# [>>> example_dict.txt <<<](https://github.com/stscoundrel/harlaw/files/7482353/example2.txt)

**Comment by stscoundrel** [comment]
Can you post the original DSL file you tried this with? Thanks

**Comment by Bleser** [comment]
```
#NAME "大БКРС (2021-10-27)"
#INDEX_LANGUAGE "Chinese"
#CONTENTS_LANGUAGE "Russian"

二连浩特
 èrliánhàotè
 [m1]Эрэн-Хото [i](городской уезд автономного района Внутренняя Монголия КНР)[/i][/m]

三大洋
 sāndàyáng
 [m1]Три океана ([i]Тихий, Атлантический и Индийский[/i])[/m]

三比西河
 sānbǐxīhé
 [m1]река Замбези ([i]Южная Африка[/i])[/m]

董里
 dǒnglǐ
 [m1]Транг ([i]провинция и город в Таиланде[/i])[/m]
```

**Comment by stscoundrel** [comment]
Your example appears to use spaces instead of tabs for indentation of word meanings. Current version of Harlaw expects spaces, as all example DSL files I had were produced that way.

According to DSL Card Structure docs spaces seem to also be allowed, so I'll add support for them later. If you need this faster than that, you can also edit your own DSL file to:

- Use tab instead of spaces
- Remove empty lines between words

Thanks for reporting this.

**Comment by stscoundrel** [comment]
Hey @Bleser, I just published v1.1.0. It should fix your issue. Try again with that version, please.

**Comment by Bleser** [comment]
@stscoundrel 
Thank you, now there is no more error.
But as it turned out, the bkrs.info dictionary contains some structural errors.
Which is why I had to write a parser for it.
